packages/vats/src/lib-board.js

```diff
@@ -59,28 +59,39 @@ function makeBoard(
     return Far(`${ifaceInaccessiblePrefix}${iface}`, {});
   };
 
-  // Create a marshaller that just looks up objects, not publish them.
-  const readonlyMarshaller = Far('board readonly marshaller', {
-    ...makeMarshal(val => {
-      if (!valToId.has(val)) {
-        // Unpublished value.
-        return null;
-      }
-
-      // Published value.
-      return valToId.get(val);
-    }, slotToVal),
+  const withStringify = m => ({
+    ...m,
+    serializeAndStringify: v => JSON.stringify(m.serialize(v)),
   });
 
+  // Create a marshaller that just looks up objects, not publish them.
+  const readonlyMarshaller = Far(
+    'board readonly marshaller',
+    withStringify({
+      ...makeMarshal(val => {
```

Comment:
I've we're going to adopt this, why not have `makeMarsal` (if not an even deeper layer) provide the composite method?

Reply:
> I'd like to get some feedback on this in draft form.

I echo @gibson042's desire for clearer names and to consider putting this into Endo instead.

IMO it's not too late to advance https://github.com/endojs/endo/issues/1248 . The new method `serializeAndStringify` could be more clear as `toCapDataAndStringify` or `stringifyAsCapData`. I'd suggest `serializeAsCapData` but that term is ambiguous now (ergo issue 1248).

Reply:
OK, cool... so pushing it down into endo is pretty clearly The Right Thing. (I wasn't aware of 1248 and I didn't know how receptive others would be to the suggestion.)

Reply:
I agree it is still not to late to rename the marshal methods. I approved https://github.com/endojs/endo/pull/1402 last year and my LGTM still stands. There is no unresolved "Request changes". 

I know from painful experience that renaming without breaking things can be painful but worth it. https://github.com/endojs/endo/pull/1402 is the right first step -- it just introduces the new names while deprecating the old.

I am open to having `makeMarshal` provide the composite methods. This makes more sense to me, but I'll of course withhold judgement until I see what it looks like. But please do https://github.com/endojs/endo/pull/1402 first, unless there's an impediment I'm not seeing.